src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn get-abort-submission-path [workspace-id submission-id]
+  (submission-details workspace-id submission-id))
+
```

Comment:
I'm just EDDIE and I've used AJAX before, but not REST extensively (I've made both front-end AJAX handler via php and back-end AJAX).  Maybe this means I am not "well RESTed"????

I just did this because it made sense to me to compensate for the effect of overloading (GET and DELETE) onto one path.